Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,44 @@
-function setAlarm() {}
+let countdown
+let timeLeft
+
+function setAlarm() {
+  const timeRemaining = document.getElementById("timeRemaining");
+  const alarmInput = document.getElementById("alarmSet");
+  timeLeft = parseInt(alarmInput.value);
+
+  if (isNaN(timeLeft) || timeLeft <= 0) {
+    alert("Please enter a valid number which must be greater than 0");
+  }
+
+  if (countdown) {
+    clearInterval(countdown);
+  }
+
+  function updateDisplay() {
+    const minutes = Math.floor(timeLeft / 60);
+    const seconds = timeLeft % 60;
+    timeRemaining.textContent = `Time Remaining: ${String(minutes).padStart(
+      2,
+      "0"
+    )}:${String(seconds).padStart(2, "0")}`;
+  }
+
+  
```

Comment:
I would encourage you to install `prettier` extension in VScode (if it is Iyour IDE) so when you save your code, it will remove spaces for you and shape your code. 
I would encourage you to remove extra spaces which are not needed.

Reply:
Thanks so much for your valuable feedback!